### PR TITLE
Pract.04 task01 03

### PR DESCRIPTION
**Pull request by Viktoria035**
Viktoria Koleva
6   5MI0600139

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task02.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+int isEven(int num)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task02.cpp:2]: Should return bool

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+void isPositive(int numberP)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:2]: Should return bool

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{
+	if (numberP >= 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:4]: zero is not positive

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	{
+		std::cout << "Your number is positive";
+	}
+	else
+	{
+		std::cout << "Your number is negative";
+	}
+	
+}
+void isNegative(int numberN)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:14]: should return bool

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Follow PR rules for PR title.

Closing due to merged solutions.
Address comments and contact us if you have questions.

**Pull request closed without merging**